rules-tests/Php80/Rector/NotIdentical/StrContainsRector/Fixture/multibyte_offset_strpos.php.inc

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Rector\Tests\Php80\Rector\NotIdentical\StrContainsRector\Fixture;
+
+class MultibyteOffsetStrpos
+{
+    public function run()
+    {
+        $isMatch = mb_strpos('happy 😊 euro €', 'a', 1) !== false;
+    }
+}
+?>
+-----
+<?php
+
+namespace Rector\Tests\Php80\Rector\NotIdentical\StrContainsRector\Fixture;
+
+class MultibyteOffsetStrpos
+{
+    public function run()
+    {
+        $isMatch = str_contains(substr('happy 😊 euro €', 1), 'a');
```

Comment:
need `mb_substr` ?

Reply:
Good catch! I updated the 2 tests and rule to handle this case